lib/vector-buffers/src/topology/builder.rs

```diff
@@ -273,7 +275,7 @@ mod tests {
     async fn single_stage_topology_block() {
         let mut builder = TopologyBuilder::<Sample>::default();
         builder.stage(
-            MemoryBuffer::new(NonZeroUsize::new(1).unwrap()),
+            MemoryBuffer::with_max_events(NonZeroUsize::new(1).unwrap()),
```

Comment:
I wonder if some of the tests shouldn't also exercise the `max_bytes` mode. Not sure…